File: rest-service/src/main/java/mb/api/controller/resources/LinkGuest.java
package mb.api.controller.resources;

import javax.ws.rs.Consumes;
import javax.ws.rs.POST;
import javax.ws.rs.Path;
import javax.ws.rs.core.MediaType;

import mb.api.model.IOGuestLink;

@Path("/link-guest")
public interface LinkGuest
{
  @POST
  @Consumes(MediaType.APPLICATION_JSON)
  void convGuestToUser(IOGuestLink link);
}

File: rest-service/src/main/kotlin/mb/api/controller/resources/LinkGuest.kt
package mb.api.controller.resources

import mb.api.model.IOGuestLink
import jakarta.ws.rs.Consumes
import jakarta.ws.rs.POST
import jakarta.ws.rs.Path
import jakarta.ws.rs.core.MediaType

@Path("/link-guest")
interface LinkGuest
{
  @POST
  @Consumes(MediaType.APPLICATION_JSON)
  fun convGuestToUser(link: IOGuestLink)
}
